### DISCUSSION TITLE
Dark mode

### DISCUSSION THREAD

**Discussion by IVIIIXV** [orbat-mapper/orbat-mapper#239] [Ideas]
can we have the option for dark mode? using the dark reader extension on chrome works ok but it messes up the symbols.

thank you for your time and all your hard work!

**Comment by kjellmf** [comment]
It is on my todo list. See issue #63 for updates. It requires a lot of work, so I probably have to add it gradually.

**Comment by kjellmf** [comment]
@IVIIIXV Any thoughts on how symbols should be rendered in dark mode? I did a quick experiment, but I feel that the standard symbol colors are a bit bright.

![image](https://github.com/user-attachments/assets/d968b8c7-21e6-41aa-b41c-addb58ffa98a)
![image](https://github.com/user-attachments/assets/40587a28-9c5e-4f66-880f-f56aa5cf6939)

**Reply by IVIIIXV** [reply]
Thanks for the follow up! I believe they are fine just like that  in general but if i had the option for a more colour muted version or even monochromatic i would always choose that, but thats just me :)

**Reply by kjellmf** [reply]
Thanks for the feedback!

**Reply by IVIIIXV** [reply]
No! Thank you for all your hard work on this project! You are doing a great job with it!

**Comment by kjellmf** [comment]
There is now a dark mode toggle available in the top navigation toolbar. There are probably still a few components that does not support dark mode properly. I'll fix them when I find them. 

<img width="1281" height="592" alt="image" src="https://github.com/user-attachments/assets/7ba1fb89-173b-43fd-b8d6-a70244d9ddff" />
